bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
     running_options.add_argument('--input_sequence_type', help='Specify whether the input sequence is "nucleotide" or "aminoacid" sequence data (default=will attempt to auto-detect)', choices = ['aminoacid', 'nucleotide'],  default=argparse.SUPPRESS)
-    running_options.add_argument('--filter_minimum', type=int, metavar='filter_minimum', help='Minimum number of positions that must be aligned for each sequence in order to by placed in the phylogenetic tree (default: %sbp for nucleotide packages, %s aa for protein packages)' %
+    running_options.add_argument('--filter_minimum', type=int, metavar='filter_minimum', help='Minimum number of positions that must be aligned for each sequence in order to be placed in the phylogenetic tree (default: %s bp for nucleotide packages, %s aa for protein packages)' %
                                  (Run.MIN_ALIGNED_FILTER_FOR_NUCLEOTIDE_PACKAGES, Run.MIN_ALIGNED_FILTER_FOR_AMINO_ACID_PACKAGES))
 
     searching_options = graft_parser.add_argument_group('searching options')
-    searching_options.add_argument('--evalue', metavar='evalue', help='evalue cutoff for the hmmsearch (default = 1e-5)', default= '1e-5')
-    searching_options.add_argument('--search_and_align_only', action="store_true", help='Stop after reads have been identified (default=False)', default=False)
-    searching_options.add_argument('--search_only', action="store_true", help='Stop after reads searched (default=False)', default=False)
-    searching_options.add_argument('--euk_check', action="store_true", help='Search whole sample for 18S (default=False)', default=False)
-    searching_options.add_argument('--search_method', choices=('hmmsearch','diamond'), help='Search method (default=hmmsearch)', default='hmmsearch')
-    searching_options.add_argument('--maximum_range', type=int, help='maximum range to use when searching for potentially linked reads (when searching contigs)', default=None)
-    searching_options.add_argument('--expand_search_contigs', nargs='+', help='generate an HMM for searching --forward data by first searching these contigs')
+    searching_options.add_argument('--evalue', metavar='evalue', help='Specify the evalue cutoff for the hmmsearch, if you would like to use a cutoff different to the default (1e-05) or the trusted cutoff (TC) within the HMM.', default= '1e-5')
+    searching_options.add_argument('--search_and_align_only', action="store_true", help='Stop GraftM running after reads have been identified and aligned (i.e. no placement step, default=False)', default=False)
+    searching_options.add_argument('--search_only', action="store_true", help='Stop GraftM running after reads have been identified (i.e. no alignment or placement steps, default=False)', default=False)
+    searching_options.add_argument('--euk_check', action="store_true", help='Cross check identified reads using an 18S specific HMM to help filter out eukaryotic ribosomal reads (default=False)', default=False)
```

Comment:
I think this flag would be better under the nucleotide-specific options, beside where you can specify the hmm